src/webapp/src/client/app/common/directives/wall/posts/post-create/post-create.component.js

```diff
@@ -66,14 +77,45 @@
             }
         }
 
+        function selectMention(item) {
+            vm.selectedMentions.push({id: item.id, fullName: item.label});
+
+            return `**@${item.label.replace(' ', '_')}**`;
```

Comment:
Missed that part and found about it only while testing..
Can we not use markdown syntax to highlight users? In edit form it's enough to have it plain, and during rendering the post - just parse the message and bold/emphasize it.